src/dswx_sar/masking_with_ancillary.py

```diff
@@ -19,6 +19,7 @@
                       region_growing)
 from dswx_sar.dswx_runconfig import _get_parser, RunConfig
 
+from memory_profiler import profile
```

Comment:
Is it your intention to include this module to the release version of SAS?

Reply:
Thank you. This should be deleted.